browser/ui/views/playlist/playlist_action_dialogs.cc

```diff
@@ -210,6 +210,16 @@ void ShowMoveItemsDialog(content::WebContents* contents,
       FindBrowserViewFromSidebarContents(contents), std::move(param));
 }
 
+void ShowPlaylistSettings(content::WebContents* contents) {
+  auto* browser_view = FindBrowserViewFromSidebarContents(contents);
+  CHECK(browser_view);
+  content::OpenURLParams open_params(GURL("brave://settings/playlist"),
+                                     content::Referrer(),
+                                     WindowOpenDisposition::NEW_FOREGROUND_TAB,
+                                     ui::PAGE_TRANSITION_AUTO_TOPLEVEL, false);
+  browser_view->browser()->OpenURL(open_params);
```

Comment:
How about using singleton tab?

Reply:
@sangwoo108 why would you choose consistency with one oddball settings page over all other settings pages?